src/v2/view-builder/views/shared/ChallengePushView.js

```diff
@@ -0,0 +1,101 @@
+// Common view for OV push and custom push.
+import { _, loc, createButton, View } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import { BaseForm } from '../../internals';
+import polling from '../shared/polling';
+import { WARNING_TIMEOUT } from '../../utils/Constants';
+import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
+import { AUTHENTICATOR_KEY } from '../../../ion/RemediationConstants';
+
+const warningTemplate = View.extend({
+  className: 'okta-form-infobox-warning infobox infobox-warning',
+  template: hbs`
+    <span class="icon warning-16"></span>
+    <p>{{warning}}</p>
+  `
+});
+const Body = BaseForm.extend(Object.assign(
+  {
+    noButtonBar: true,
+
+    className() {
```

Comment:
This doesnt work due to https://github.com/okta/okta-ui/blob/master/packages/courage/src/views/forms/BaseForm.js#L195

Reply:
update the className in `initialize()` ??